source/fe/mapping_c1.cc

```diff
@@ -66,7 +66,7 @@ MappingC1<2>::MappingC1Generic::add_line_support_points (const Triangulation<2>:
                                                          std::vector<Point<2> > &a) const
 {
   const unsigned int dim = 2;
-  std::vector<Point<dim> > line_points (2);
+  constexpr std::array<double, 2> interior_gl_points {{0.5 - std::sqrt(1.0/20.0), 0.5 + std::sqrt(1.0/20.0)}};
```

Comment:
Hm, is this correct? According to https://en.wikipedia.org/wiki/Gaussian_quadrature, the interior locations for the 4-point rule are `\pm \sqrt{1/5}` on `[-1,1]`, which would translate to `0.5 \pm 0.5*\sqrt{1/5}` -- which is, well, exactly what you have here. :-))

In other words, could I convince you to maybe write it as `0.5 \pm 0.5*\sqrt{1/5}`?

Reply:
Absolutely; I just copied and pasted from somewhere else.